core/modules/user/tests/user.test

```diff
@@ -2053,7 +2053,7 @@ class UserSignatureTestCase extends BackdropWebTestCase {
     preg_match('/#comment-([0-9]+)/', $this->getURL(), $match);
     $comment_id = $match[1];
 
-    // Log in as an administrator and edit the comment to use Full HTML, so
+    // Log in as an administrator and edit the comment to use Raw HTML (full_html), so
     // that the comment text itself is not filtered at all.
```

Comment:
```suggestion
    // Log in as an administrator and edit the comment to use Raw HTML
    // (full_html), so that the comment text itself is not filtered at all.
```